homeassistant/components/wallbox/const.py

```diff
@@ -20,80 +24,87 @@
 
 CONF_SENSOR_TYPES = {
     "charging_power": {
-        CONF_ICON: "mdi:ev-station",
+        CONF_ICON: None,
         CONF_NAME: "Charging Power",
         CONF_ROUND: 2,
         CONF_UNIT_OF_MEASUREMENT: POWER_KILO_WATT,
-        STATE_UNAVAILABLE: False,
+        CONF_DEVICE_CLASS: DEVICE_CLASS_POWER,
     },
     "max_available_power": {
-        CONF_ICON: "mdi:ev-station",
+        CONF_ICON: None,
         CONF_NAME: "Max Available Power",
         CONF_ROUND: 0,
         CONF_UNIT_OF_MEASUREMENT: ELECTRICAL_CURRENT_AMPERE,
```

Comment:
ELECTRIC_CURRENT_AMPERE

Reply:
Thnx for the review, and, good catch. ELECTRICAL apparently has become ELECTRIC in the past weeks ;-)